### ISSUE TITLE
unnamed-scenarios doesn't work for Scenario Outline

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#155]
Hi!

It does not report an error regardless Scenario Outline does not have a name.
``` gherkin
Feature: Feature with description

Scenario Outline:
  Then this is a then step
Examples:
  | foo |
  | bar |
```
